modules/core/nuxeo-core-el/src/main/java/org/nuxeo/ecm/platform/el/Functions.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Thomas Roger <troger@nuxeo.com>
+ */
+
+package org.nuxeo.ecm.platform.el;
+
+import java.util.Arrays;
+
+/**
+ * Helper functions injected in the {@link ExpressionContext} instance.
+ *
+ * @since 11.2
+ */
+public class Functions {
+
+    /**
+     * Returns {@code true} if the given {@code arr} contains the specified element.
```

Comment:
For consistency:
Returns {@code true} if the given {@code arr} contains the given {@code o}.
or
Returns {@code true} if the given array contains the given object.

Reply:
You should specify in the Javadoc what happens for a `null` array (it's fine to make it behave like an empty one).